Gemfile

```diff
@@ -50,6 +54,10 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  # Shoulda Matchers. Read more: https://github.com/thoughtbot/shoulda-matchers
+  gem 'shoulda-matchers', '~> 4.0'
+  # Database cleaner. Read more: https://github.com/DatabaseCleaner/database_cleaner
+  gem 'database_cleaner-active_record'
```

Comment:
I see you've installed this gem, but it should be configured in `spec_helper.rb`.
Read more 
https://github.com/DatabaseCleaner/database_cleaner#how-to-use
https://avdi.codes/configuring-database_cleaner-with-rails-rspec-capybara-and-selenium/

Reply:
Thanks. I configured database_cleaner according mentioned guide.